privacy_estimates/binomial_proportion.py

```diff
@@ -3,13 +3,12 @@
 from statsmodels.stats.proportion import proportion_confint
 
 from privacy_estimates.utils import AttackResults
-from privacy_estimates.privacy_region import eps_from_fnr_fpr
+from privacy_estimates.privacy_region import eps_from_fnr_fpr, delta_from_fnr_fpr
 
 
-def compute_eps_lo_hi(count: AttackResults, delta: float, alpha: float, method: str) -> Tuple[float, float]:
+def compute_binomial_proportion(count: AttackResults, alpha: float, method: str) -> Tuple[float, float, float, float]:
```

Comment:
I don't think the naming is descriptive enough. Making `proportion` plural sounds slightly better for me.

Reply:
I agree. I propose we change to `compute_confidence_region`. How does that sit?